src/main/scala/scalm/http/package.scala

```diff
@@ -0,0 +1,17 @@
+package scalm
+
+import scala.language.implicitConversions
+
+package object http {
+
+  type Decoder[A] = String => Either[String, A]
+
+  implicit def methodToString(m: Method): String =
```

Comment:
Maybe could be better use just `method.toString.toUpperCase` above where it is needed and save this implicit conversion. I mean it seems that would be used just in that place.

Reply:
Yes, that makes sense.